kubernetes/rollup-chart/files/scripts/start-geth.sh

```diff
@@ -2,6 +2,9 @@
 
 set -o errexit -o nounset
 
+export CHAIN_ID=$evm_chain_id
```

Comment:
This envar is already set in the deployment.yml, but I don't think that's the best thing to do since it's easy to miss.

I wanted to keep casing consistent in our configMaps, but I'd rather them all be UPPERCASE like envars usually are. That would get rid of the need for these exports.